packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```diff
@@ -64,3 +64,45 @@
     }
   }
 }
+
+.bpk-sticky-chip {
+  display: flex;
+  align-items: center;
+  transition: gap tokens.$bpk-duration-sm ease;
+  fill: currentcolor;
+  gap: tokens.bpk-spacing-md();
+
+  &--collapsed {
+    transition: padding tokens.$bpk-duration-sm ease;
+    gap: 0;
+    padding-inline: tokens.bpk-spacing-md();
+
+    .bpk-sticky-chip {
+      gap: 0;
+    }
+  }
+
+  &--expanded {
+    transition: padding tokens.$bpk-duration-sm ease;
+  }
+
+  &--text {
+    display: inline-block;
+    max-width: tokens.$bpk-modal-max-width;
```

Comment:
Setting `max-width` to this size is a bit weird. You probably just want to override the `max-width` in the animation, so it might make more sense to change it to `max-width: unset`.